File: Data Structures & Algorithms/subsets/submission-0.py
class Solution:
    def subsets(self, nums: List[int]) -> List[List[int]]:
        res=[]
        def s_s(i,new_arr):
            if i==len(nums):
                
                res.append(new_arr)
                return 
            
            s_s(i+1,new_arr+[nums[i]])
            s_s(i+1,new_arr)

        s_s(0,[])

        return res
        

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {

       int n=nums.size();
       unordered_set<int> s(nums.begin(),nums.end());
       int max_count=0;
       for(int num:s)
       {
        int current=num;
        int count =1;
        while(s.find(current+1)!=s.end())
        {
            current++;
            count++;
        }
        max_count=max(count,max_count);
       }
       return max_count;
    }
};
